api/src/main/java/jakarta/security/auth/message/AuthException.java

```diff
@@ -17,13 +18,15 @@
 
 package jakarta.security.auth.message;
 
+import javax.security.auth.login.LoginException;
+
 /**
  * A generic authentication exception.
  *
  */
-public class AuthException extends javax.security.auth.login.LoginException {
+public class AuthException extends LoginException {
 
-    private static final long serialVersionUID = -1156951780670243758L;
+    private static final long serialVersionUID = 1560420530928138346L;
```

Comment:
It looks like backward-compatible change, so why modify this?

Reply:
I regenerated it, to see if it would change. I'm not actually such an expert in the serialVersionUID usages. Of course the state didn't change though. I'll change it back. Thx!